docs/rfcs/0000-aligned-timestamp.md

```diff
@@ -0,0 +1,68 @@
+# How to Align Timestamps
+
+Strongly define how time buckets are aligned, specifically with an half-open interval
+`[start, end)`.
+
+* **Status** Draft
+* **Author** John-John Tedro <udoprog@spotify.com>
+* **PR** https://github.com/spotify/heroic/pull/208
+
+## Requirements
+
+The output buckets of sampling aggregations must fall within the captured interval of the same bucket in the next sampling aggregation.
+This avoids samples 'travelling' when chaining multiple sampling aggregations, so that `a | b` is equivalent to `a | a | b`. A concrete example is that `spread | sum` must result in the same buckets as just `sum` to allow for correctly functioning global aggregations.
+
+Current fetches for `BigtableBackend` have one edge case for every period interval where data is invisible (see: https://github.com/udoprog/heroic/blob/fake-bigtable-backend/heroic-test-it/src/main/java/com/spotify/heroic/test/AbstractMetricBackendIT.java#L129).
+An interval definition should be picked which makes this easier to avoid.
+
+## Suggestion
+
+The current time buckets have values at timestamp `t` include the range `(t - extent, t]` in each
+bucket.
+
+Define a `DateRange` to be `[start, end)`. Buckets at timestamp `t`, should include data in the
```

Comment:
Does this refer to a class name? If so, maybe better to keep it outside the RFC to avoid going into implementation.